app/renderer/js/main.ts

```diff
@@ -239,6 +240,14 @@ class ServerManagerView {
 		this.toggleSidebar(showSidebar);
 	}
 
+	/*  Removes the stale UA string from the disk if the app is not freshly installed
+		We should remove this in a further release	*/
+	removeUAfromDisk(): void {
+		if (ConfigUtil.isConfigItemExists('userAgent')) {
```

Comment:
Do we need this conditional? I believe the db handles that for us, doesn't it?

Reply:
Probably it does. I was a little unsure about that. But you confirmed it for me. Thanks!